st2common/st2common/util/pack.py

```diff
@@ -102,6 +104,14 @@ def validate_config_against_schema(config_schema, config_object, config_path,
 
     pack_name = pack_name or 'unknown'
 
+    for key in config_object:
+        if (jinja_utils.is_jinja_expression(value=config_object.get(key)) and
+                "decrypt_kv" in config_object.get(key) and config_schema.get(key).get('secret')):
+            raise ValueValidationException('Values specified as secret: True in config schema '
+                                           'are automatically decrypted by default. Use of '
+                                           'decrypt_kv jinja filter is not allowed for such '
+                                           'values.')
```

Comment:
I should be more specific. Can you use double quotes instead?

```
Values specified as "secret: True" in the config schema are automatically decrypted by default. Use of the "decrypt_kv" jinja filter is not allowed for such values.
```

Reply:
Done